README.md

```diff
@@ -1,7 +1,7 @@
-# Codex.Editor server validation sample
+# EditorJS PHP server validation
```

Comment:
Editor.js PHP

Server-side implementation sample for the [Editor.js](). Contains validation and HTML sanitising logic and converts editor's JSON output to the useful Block objects.